packages/_debian-common/changelog

```diff
@@ -1,9 +1,12 @@
-libsuperiormysqlpp (0.3.3) UNRELEASED; urgency=medium
+libsuperiormysqlpp (0.3.4) unstable; urgency=medium
```

Comment:
I would leave UNRELEASED here, if you take look at master commits, you will find release commits, that changes this line.

Reply:
ACK